src/Uno.UI/UI/Xaml/Controls/Icon/FontIconSource.cs

```diff
@@ -0,0 +1,64 @@
+using Windows.UI.Text;
+using Windows.UI.Xaml.Media;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class FontIconSource : global::Windows.UI.Xaml.Controls.IconSource
```

Comment:
We have also the MUX version of these here - `Uno.UI\Microsoft\UI\Xaml\Controls\IconSource`, maybe the code could be copied from there for the other icon types too

Reply:
Actually the code is automatically "copied" to MUX (that's why my build failed :) )